### PR TITLE
Move nowcast parameters to constants file and add placeholders

### PR DESCRIPTION
**Pull request by chinandrew**
### Description
Moving relevant constants for nowcasting to their own file and adding a test that duplicates each value and will fail if any are changed. This is to ensure changes to parameters are done explicitely. Values are placeholders for now.

cc @mariajahja 

### Changelog
Itemize code/test/documentation changes and files added/removed.
- Add parameters to a constants files and use them instead of function default parameters
- update tests to reflect lack of default constants
- Change export_data bool to export_dir string to prepare for putting export _dir into the params file and passing it in after read_params() is called.

### CODE REVIEW

File: nowcast/tests/test_constants.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"""This test is to ensure the constants are not accidentally altered."""
```
Comment (krivard) [inline nowcast/tests/test_constants.py:1]: This is a little bit [Frog And Toad Try Not To Eat Cookies](https://sites.fas.harvard.edu/~fs22k/Assignments/frog-toad.html) but I'm not sure there's a better way to do it so 🤷

**Review by krivard — APPROVED** [review]
👍

**Comment by krivard** [comment]
(modulo the linting fix)

File: nowcast/tests/test_constants.py (hunk 1 above)
Comment (chinandrew) [reply nowcast/tests/test_constants.py:1]: i guess what we're protecting against is conceptually no different than someone changing the implementation of smoothing or something and us not catching it, which an output validation integration test would capture, so we could move to that once it's ready.

**Comment by chinandrew** [comment]
> (modulo the linting fix)

fixed
